doc/man1/asn1parse.pod

```diff
@@ -173,11 +173,11 @@ Parse a DER file:
 
 Generate a simple UTF8String:
 
- openssl asn1parse -genstr 'UTF8:Hello World'
+ openssl asn1parse -genstr "UTF8:Hello World"
```

Comment:
My opposition aside, this particular change isn't about contractions, and isn't necessary.